src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -2326,7 +2328,7 @@ protected void Log(string msg, WarningLevel severity)
                     Console.WriteLine(args.ErrorContext.Error);
                 },
                 ReferenceLoopHandling = ReferenceLoopHandling.Ignore,
-                TypeNameHandling = TypeNameHandling.Auto,
+                TypeNameHandling = TypeNameHandling.None,
```

Comment:
I see, you've made this change in general for all nodes. How then does dyn deserialization work if the type information is missing?

Reply:
I've left this set to Auto when the settings are used for serialization. The types names should still be present for nodes, though I don't believe we need auto handling for deserialization because of our `nodeReadConverter` - I will manually test this with some explicit Node Model nodes, but I am hoping the test suite is robust enough to catch this if it's an issue.

Reply:
@aparajit-pratap I've confirmed UI nodes load correctly and dummy nodes function as well for resolving.